converters/fromZigbee.js

```diff
@@ -2170,6 +2170,10 @@ const converters = {
                 const reversalLookup = {0: 'OFF', 1: 'ON'};
                 result.motor_reversal = reversalLookup[value];
             }
+            if (msg.data.hasOwnProperty('moesCalibrationTime')) {
+                const value = parseFloat(msg.data['moesCalibrationTime']) / 10.0;
+                result.calibration_time = value;
+            }
```

Comment:
> where it was not possible to calibrate the motor running time

I don't see how this solves that problem, this just exposes the `calibration_time` to MQTT but there is no way to set it.

Reply:
Yeah, sorry - wrong choice of words on my side. Yes, it just exposes the calibration_time, setting is implemented via toZigbee.js